### PR TITLE
Issue 73

### LINKED ISSUE #73: [NTHORDER] NTHOrder - Module fails to Initialize
**Module Information**
* Identifier: NTHORDER
* Module Name: Warlords of Order & Chaos
* Version: unknown

**Describe the bug**
Module fails to initialize

**To Reproduce**
1. Add module to moduleConfig.json

**Expected behavior**
Module initializes

**Screenshots**
```
Critical Exception has occured:
System.ArgumentException: An item with the same key has already been added. Key: STAYCAS
   at System.Collections.Generic.Dictionary`2.TryInsert(TKey key, TValue value, InsertionBehavior behavior)
   at MBBSEmu.Module.MsgFile.BuildMCV()
   at MBBSEmu.Module.MsgFile..ctor(String modulePath, String msgName)
   at MBBSEmu.Module.MbbsModule..ctor(IFileUtility fileUtility, String moduleIdentifier, String path, MemoryCore memoryCore)
   at MBBSEmu.Program.Run(String[] args)
```

**Software Information:**
- OS: Windows Server 2019 Standard
- Telnet Client: N/A

### PR DESCRIPTION
**Pull request by enusbaum**
Fixes #73 and also adds required ordinals to be able to enter the game

### CODE REVIEW

File: MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs [hunk 1]
```diff
@@ -7240,5 +7254,110 @@ private void strlwr()
...
+            
+            //Truncate to 29 bytes
+            if(returnedWord.Length > 29)
+                returnedWord.SetLength(29);
+
+            returnedWord.WriteByte(0);
+            
+            Module.Memory.SetArray(returnPointer, returnedWord.ToArray());
+
+#if DEBUG
```
Comment (paladine) [inline MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs:7344]: With exhaustive unit tests, do we need the log spam?

File: MBBSEmu.Tests/ExportedModules/Majorbbs/cncwrd_Tests.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class cncwrd_Tests : MajorbbsTestBase
+    {
+        private const int CNCWORD_ORDINAL = 130;
+
+        [Theory]
+        [InlineData("FIRST SECOND\0", 0, "FIRST", 6)]
+        [InlineData("FIRST SECOND\0", 6, "SECOND", 13)]
+        [InlineData("FIRST SECOND THIRD\0", 1, "IRST", 6)]
```
Comment (paladine) [inline MBBSEmu.Tests/ExportedModules/Majorbbs/cncwrd_Tests.cs:15]: how about a test that moves from SECOND -> THIRD cleanly?

**Review by paladine — CHANGES_REQUESTED** [review]

File: MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs (hunk 1 above)
Comment (enusbaum) [reply MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs:7344]: Probably not -- they're usually more in there as a tracer on when things would go wrong earlier on with Memory management. But you're right, probably don't need the obvious so verbose now 👍

File: MBBSEmu.Tests/ExportedModules/Majorbbs/cncwrd_Tests.cs (hunk 2 above)
Comment (enusbaum) [reply MBBSEmu.Tests/ExportedModules/Majorbbs/cncwrd_Tests.cs:15]: Added! 👍

**Review by paladine — APPROVED** [review]

**Pull request merged**